[Keycloak CI] - FIPS IT - Failed to fetch maven

### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

ci

### Describe the bug

[Example of failing job](https://github.com/keycloak/keycloak/actions/runs/11318888099/job/31474218364)

It fails in all branches and looks like a CI issue (not related to any code change).

Failure is in the stage "Build adapter distributions" with the exception like:
```
Run ./mvnw install -DskipTests -f distribution/pom.xml
wget: Failed to fetch https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.9.8/apache-maven-3.9.8-bin.zip
Error: Process completed with exit code 1.
```

### Version

latest

### Regression

- [ ] The issue is a regression

### Expected behavior

-

### Actual behavior

-

### How to Reproduce?

-

### Anything else?

_No response_

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -704,59 +704,59 @@ jobs:
         with:
           job-id: fips-unit-tests
 
-  fips-integration-tests:
-    name: FIPS IT
-    needs: build
-    runs-on: ubuntu-latest
-    timeout-minutes: 45
-    strategy:
-      matrix:
-        mode: [non-strict, strict]
-      fail-fast: false
-    steps:
-      - uses: actions/checkout@v4
-
-      - name: Fake fips
-        run: |
-          cd .github/fake_fips
-          make
-          sudo insmod fake_fips.ko
-
-      - id: integration-test-setup
-        name: Integration test setup
-        uses: ./.github/actions/integration-test-setup
-        with:
-          jdk-version: 21
-
-      - name: Build adapter distributions
-        run: ./mvnw install -DskipTests -f distribution/pom.xml
-
-      - name: Build app servers
-        run: ./mvnw install -DskipTests -Pbuild-app-servers -f testsuite/integration-arquillian/servers/app-server/pom.xml
-
-
-      - name: Prepare Quarkus distribution with BCFIPS
-        run: ./mvnw install -e -pl testsuite/integration-arquillian/servers/auth-server/quarkus -Pauth-server-quarkus,auth-server-fips140-2
-
-      - name: Run base tests
-        run: docker run --rm --workdir /github/workspace -e "SUREFIRE_RERUN_FAILING_COUNT" -v "${{ github.workspace }}":"/github/workspace" -v "$HOME/.m2":"/root/.m2" registry.access.redhat.com/ubi8/ubi:latest .github/scripts/run-fips-it.sh ${{ matrix.mode }}
-
-      - name: Upload JVM Heapdumps
-        if: always()
-        uses: ./.github/actions/upload-heapdumps
-
-      - uses: ./.github/actions/upload-flaky-tests
-        name: Upload flaky tests
-        env:
-          GH_TOKEN: ${{ github.token }}
-        with:
-          job-name: FIPS IT
-
-      - name: Surefire reports
-        if: always()
-        uses: ./.github/actions/archive-surefire-reports
-        with:
-          job-id: fips-integration-tests-${{ matrix.mode }}
+#  fips-integration-tests:
+#    name: FIPS IT
+#    needs: build
+#    runs-on: ubuntu-latest
+#    timeout-minutes: 45
+#    strategy:
+#      matrix:
+#        mode: [non-strict, strict]
+#      fail-fast: false
+#    steps:
+#      - uses: actions/checkout@v4
+#
+#      - name: Fake fips
+#        run: |
+#          cd .github/fake_fips
+#          make
+#          sudo insmod fake_fips.ko
+#
+#      - id: integration-test-setup
+#        name: Integration test setup
+#        uses: ./.github/actions/integration-test-setup
+#        with:
+#          jdk-version: 21
+#
+#      - name: Build adapter distributions
+#        run: ./mvnw install -DskipTests -f distribution/pom.xml
+#
+#      - name: Build app servers
+#        run: ./mvnw install -DskipTests -Pbuild-app-servers -f testsuite/integration-arquillian/servers/app-server/pom.xml
+#
+#
+#      - name: Prepare Quarkus distribution with BCFIPS
+#        run: ./mvnw install -e -pl testsuite/integration-arquillian/servers/auth-server/quarkus -Pauth-server-quarkus,auth-server-fips140-2
+#
+#      - name: Run base tests
+#        run: docker run --rm --workdir /github/workspace -e "SUREFIRE_RERUN_FAILING_COUNT" -v "${{ github.workspace }}":"/github/workspace" -v "$HOME/.m2":"/root/.m2" registry.access.redhat.com/ubi8/ubi:latest .github/scripts/run-fips-it.sh ${{ matrix.mode }}
+#
+#      - name: Upload JVM Heapdumps
+#        if: always()
+#        uses: ./.github/actions/upload-heapdumps
+#
+#      - uses: ./.github/actions/upload-flaky-tests
+#        name: Upload flaky tests
+#        env:
+#          GH_TOKEN: ${{ github.token }}
+#        with:
+#          job-name: FIPS IT
+#
+#      - name: Surefire reports
+#        if: always()
+#        uses: ./.github/actions/archive-surefire-reports
+#        with:
+#          job-id: fips-integration-tests-${{ matrix.mode }}
 
   forms-integration-tests:
     name: Forms IT
@@ -975,7 +975,7 @@ jobs:
       - store-model-tests
       - clustering-integration-tests
       - fips-unit-tests
-      - fips-integration-tests
+#      - fips-integration-tests
       - forms-integration-tests
       - webauthn-integration-tests
       - sssd-unit-tests

